apps/web/.env.example

```
@@ -24,3 +24,9 @@ TELEGRAM_BOT_TOKEN=""
 
 # n8n Webhook URL (공시 폴링 및 AI 분석)
 N8N_WEBHOOK_URL=""
+
+# Sentry (에러 트래킹)
+NEXT_PUBLIC_SENTRY_DSN=""
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)